app/javascript/react/components/Square.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import LightSquareImage from "../../../assets/images/light-square.png"
+import DarkSquareImage from "../../../assets/images/dark-square.png"
```

Comment:
this might work just fine, especially if these images can be made the same size. But there may come a point where it would be easier for you to just make the black and white squares yourself using CSS, because you'll have to show an image of a piece OVER a white and black background anyways